tensorrt_llm/serve/generator_ipc.py

```
@@ -0,0 +1,1016 @@
+            await self._send(
+                identity,
+                IpcEnvelope(
+                    request_id,
+                    "stream_end",
+                    sequence,
+                    RequestOutputSnapshot.capture(promise),
+                ),
+            )
+        except asyncio.CancelledError:
+            raise
+        except Exception as error:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)